sdk/core/azure-core/azure/core/_pipeline_client_async.py

```diff
@@ -175,10 +174,10 @@ def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
         return AsyncPipeline(transport, policies)
 
     async def _make_pipeline_call(self, request, **kwargs):
-        rest_request, request_to_run = _prepare_request(request)
+        rest_request = hasattr(request, "content")
```

Comment:
Same here.

Reply:
still need to know if it's a new request, because have to convert old response to new response